### PR TITLE
Add standard when overriding existing app

### PR DESCRIPTION
**Pull request by Almouro**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/override-existing-app/project-standards/project-success/production.s.md">production.s.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: project-standards/project-success/production.s.md [hunk 1]
```diff
@@ -14,6 +14,28 @@
 - [ ] For micro-services, check each and every API when you launch in production.
 - [ ] When developing, always think about the retro-compatibility of your changes in the back-end (versioning) to make sure that people who do not have the last version of the app don't get un-wanted bugs.
 
+## When the final goal is to override an existing app
```
Comment (yleflour) [inline project-standards/project-success/production.s.md]: @Almouro Shouldn't this be in its own file?

**Review by yleflour — DISMISSED** [review]

**Review by yleflour — DISMISSED** [review]
@Almouro Shouldn't this be in its own file?

File: project-standards/project-success/production.s.md (hunk 1 above)
Comment (xavierlefevre) [inline project-standards/project-success/production.s.md]: @yleflour @Almouro , yes je suis d'accord, c'est pas directement lié au standard de projet de Mise en prod, si ? C'est plus une solution technique propre là ?

**Review by xavierlefevre — DISMISSED** [review]
@yleflour  @Almouro , yes je suis d'accord, c'est pas directement lié au standard de projet de Mise en prod, si ? C'est plus une solution technique propre là ?

File: project-standards/project-success/production.s.md (hunk 1 above)
Comment (tychota) [reply project-standards/project-success/production.s.md]: +1 !!

**Comment by Almouro** [comment]
Not sure what to set for good and bad examples. I added real stories instead

**Comment by tychota** [comment]
Need inclusion in the SUMARY.MD.

The file name should be overriding-existing-app.s.md (**s.md**, not **.md**)

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/override-existing-app/react-native/setup/overriding-existing-app.s.md">overriding-existing-app.s.md</a>**: Title doesn't contain `[Standard]` tag at the beginning
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/override-existing-app/react-native/setup/overriding-existing-app.s.md">overriding-existing-app.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/override-existing-app/react-native/setup/overriding-existing-app.s.md">overriding-existing-app.s.md</a>**: You failed to mention a *Good Examples* 
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by tychota — APPROVED** [review]

**Pull request merged**